README.md

```diff
@@ -2,4 +2,24 @@
 
 This repository contains wallpapers for use in rapps.
 
-- `data/` ― ReactOS Standard Wallpapers
+You can download these wallpapers from the "ReactOS Standard Wallpapers" package in the ReactOS Applications Manager.
+
+|File Name                          | Author                                        | Origin                                                     |
```

Comment:
add a one-space margin to the left of "file name" and the names  (like you have for "Author" and "Origin")

Reply:
Fixed.